tutorials/elixir-phoenix-on-kubernetes-google-container-engine/index.md

```diff
@@ -0,0 +1,444 @@
+
+Now you're ready to deploy your application to Container Engine!
+
+### Building in the cloud
+
+To deploy the app, you will use the
+[Google Cloud Container Build](https://cloud.google.com/container-builder/)
+service to build your Docker image in the cloud and store the resulting Docker
+image in your project in the
+[Google Cloud Container Registry](https://cloud.google.com/container-registry/).
+
+Execute the following to run the build:
+
+    gcloud container builds submit --tag=gcr.io/${PROJECT_ID}/hello:v1 .
+
+Replace `${PROJECT_ID}` with the ID of your Google Cloud Platform project.
+The period at the end is required.
+
+After the build finishes, the image `gcr.io/${PROJECT_ID}/hello:v1` will be
+available. You may list the images you have built in your project using:
+
+    gcloud container images list
+
+You may even push and pull the image directly from your registry. See the
+[Container Registry how-to guides](https://cloud.google.com/container-registry/docs/pushing-and-pulling)
+for more details.
+
+### Creating a cluster
+
+Container Engine lets you create Kubernetes clusters to host your application.
+These are clusters of VMs in the cloud, managed by a Kubernetes server.
+
+1.  Choose a cluster name. For the rest of these instructions, I'll assume that
+    name is "hello-cluster".
+
+2.  Create the cluster.
+
+        gcloud container clusters create hello-cluster --num-nodes=2
+
+    This command creates a cluster of two machines. You may choose a different
+    size, but two is a good starting point.
+
+    It may take several mintues for the cluster to be created. You can check
+    the cloud console at http://cloud.google.com/console, under the Container
+    Engine section, to see that your cluster is running. You will also be able
+    to see the individual running VMs under the Compute Engine section. Note
+    that once the cluster is running, you will be charged for the VM usage.
+
+3.  Configure gcloud to use your cluster by default, so you don't have to
+    specify it every time for the remaining gcloud commands.
+
+        gcloud config set container/cluster hello-cluster
+
+    Replace the name if you named your cluster differently.
+
+### Deploying to the cluster
+
+A production deployment comprises two parts: your Docker container, and a
+front-end load balancer (which also provides a public IP address.)
+
+We'll assume that you built the image to `gcr.io/${PROJECT_ID}/hello:v1` and
+you've created the Kubernetes cluster as described above.
+
+1.  Create a deployment
+
+        kubectl run hello-web --image=gcr.io/${PROJECT_ID}/hello:v1 --port 8080
+
+    This will run your image on a Kubernetes pod, which is the deployable unit
+    in Kubernetes. The pod opens port 8080, which is the port your Phoenix
+    application is listening on.
+
+    You may view the running pods using:
+
+        kubectl get pods
+
+2.  "Expose" the application by creating a load balancer pointing at your pod.
```

Comment:
Double quotes don't seem necessary, here and all following instances except for the ref to "v2" at 366

Reply:
done